doc/developers/contributing.rst

```diff
@@ -929,7 +929,7 @@ E.g., below is a custom classifier. For more information on this example, see
 
 get_params and set_params
 -------------------------
-All sklearn estimator have ``get_params`` and ``set_params`` functions.
+All scikit-learn estimator have ``get_params`` and ``set_params`` functions.
```

Comment:
Good call. I'll update that in a minute when I'm back at the computer.

On Aug 27, 2016 3:06 PM, "Nelson Liu" <notifications@github.com> wrote:

> In doc/developers/contributing.rst
> <https://github.com/scikit-learn/scikit-learn/pull/7268#discussion_r76521753>
> :
>
> > @@ -929,7 +929,7 @@ E.g., below is a custom classifier. For more information on this example, see
> >
> >  get_params and set_params
> >  -------------------------
> > -All sklearn estimator have ``get_params`` and ``set_params`` functions.
> > +All scikit-learn estimator have ``get_params`` and ``set_params`` functions.
>
> might as well fix this while youre at it, but estimator -> estimators
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/7268/files/0aae11c34cc0de848b2d595cc8aa525ac1cc0bc2#r76521753>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGsSVQT6uTlyIMGakfDURtuAB0tBwBXTks5qkJjPgaJpZM4JuxV9>
> .
>